Header header
bool continuous_dumpster
bool emergency_stop
float32 lateral_dis
float32 side_offset
float32 object_width
float32 left_gap
float32 right_gap